src/SFA.DAS.QnA.Application/Commands/Files/UploadFile/SubmitPageOfFilesHandler.cs

```diff
@@ -161,7 +160,7 @@ private void UpdateApplicationData(SubmitPageOfFilesRequest request, Data.Entiti
         {
             if (application != null)
             {
-                var applicationData = JObject.Parse(application.ApplicationData ?? "{}");
+                var applicationData = JsonObject.Parse(application.ApplicationData ?? "{}");
```

Comment:
Same thing here, we can cast this to a `JsonObject` instead of doing it twice below